bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -41,8 +51,65 @@
 	<channel-group-type id="general">
 		<label>General Data</label>
 	</channel-group-type>
-	<channel-group-type id="loadpoint">
-		<label>Loadpoint</label>
+	<channel-group-type id="loadpoint0">
+		<label>Loadpoint 0</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint1">
+		<label>Loadpoint 1</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint2">
+		<label>Loadpoint 2</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint3">
+		<label>Loadpoint 3</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint4">
+		<label>Loadpoint 4</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint5">
+		<label>Loadpoint 5</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint6">
+		<label>Loadpoint 6</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint7">
+		<label>Loadpoint 7</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint8">
+		<label>Loadpoint 8</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint9">
+		<label>Loadpoint 9</label>
+	</channel-group-type>
+	<channel-group-type id="loadpoint0current">
+		<label>Loadpoint 0: Current Vehicle/Heating</label>
```

Comment:
Can you possibly make these label shorter preferably < 23 characters and 2-3 words. 
The Description field can be used for more lengthy details.

Reply:
Also re run the i18n plugin to refresh the properties file